docs/ADR-48-locking-collections-in-the-builder.md

```diff
@@ -0,0 +1,63 @@
+# Locking Collections in the builder
+
+## Abstract
+
+The builder allows users to create items which are composed of GLB or PNG files, and group them in Collections. There's a distinction here between Decentraland Collections and Third Party Collections, we'll point out the differences as they arise.
+Managing items and collections is done off-chain until you decide to publish. This means that you can take items in and out of collections, modify their data, etc and it will be saved in our database. Once the publication occurs, some of the data (like the Decentraland's collection name for example) cannot change, and will be stored both in the blockchain and in our database
+
+Publications vary for each type described above:
+
+- Decentraland: You publish the entire collection with all it's items
+- Third Party: You publish each item individually, but you can do it in batches
+
+## Problem
+
+Publishing is done by sending a meta-transaction. This means that the user will send a transaction and send it to our server where it will ultimately be sent to the corresponding Polygon network to be processed.
```

Comment:
I may say `sending a transaction`. It is the same if it is a meta or normal tx

Reply:
If we're talking about meta-transactions, I would use craft here instead of repeating `send`.
```suggestion
Publishing is done by sending a meta-transaction. This means that the user will craft a transaction and send it to our server where it will ultimately be sent to the corresponding Polygon network to be processed.
```